elixir: add page (#4693)
tldr-pages/tldr: pages/common/elixir.md

# elixir

> Elixir programming language interpreter.

> > More information: <https://elixir-lang.org/>.

Reviewer: ```suggestion
> More information: <https://elixir-lang.org>.
```

- Run an Elixir file:

`elixir {{path/to/file}}`

> - Evaluate Elixir by passing it in the command:

Reviewer 2: Perhaps this will read better?
```suggestion
- Evaluate Elixir code by passing it as an argument:
```

Reviewer: @sbrl he copied the example description from the `node` page

Reviewer 2: Ah, I see. I think this makes more sense though. Maybe we should update the `node` page in a separate PR :thinking:

Author: I did. It feels like there are _many_ ways we could word this, and that it might be better to focus on what someone using `tldr` to go from one language to another might expect.

I'm more than happy to change this, certainly if we want to move toward a more canonical expression for these interpreters (e.g. `ruby`, `node`, `python`).

Reviewer: @Schoonology please apply the suggestion from @sbrl and send another PR to edit the node page with this grammar.
